ten-simple-rules-dockerfiles.Rmd

```diff
@@ -562,70 +561,64 @@ RUN pip install \
-Ignoring data files also speeds up the build in cases where there are very large files or many small files; for example, large data files or the `.git` folder of the [Git version control system](https://git-scm.com/), which contains many small files across numerous folders, are usually not needed during image build.
-In any case, you should include dummy or test data into the image to ensure that a container is functional without the actual dataset, e.g., for automated tests, instructions in the user manual, or peer review (see also "functional testing logic" in @gruening_recommendations_2019).
-For all these cases, you should provide clear instructions for users in the `README` on how to obtain actual or dummy data.
-When publishing your workspace, e.g., on Zenodo, having data and script contents as regular files outside of the container makes them more accessible to others, for example for reuse or analysis.
+Ignoring data files also speeds up the build in cases where there are very large files or many small files.
+You should include dummy or test data in the image to ensure that a container is functional without the actual dataset, e.g., for automated tests, instructions in the user manual, or peer review (see also "functional testing logic" in @gruening_recommendations_2019).
+For all these cases, you should provide clear instructions for users in the `README` file on how to obtain the actual or dummy data.
+When publishing your workspace, e.g., on Zenodo, having datasets outside of the container makes them more accessible to others, for example for reuse or analysis.
 
 A mount can also be used to access _output data_ from a container; this can be an extra mount or the same `data` directory.
 Alternatively, you can use the [`docker cp`](https://docs.docker.com/engine/reference/commandline/cp/) command to access files from a running or stopped container, but this requires a specific handling, e.g., naming the container when starting it or using multiple shells, which requires very detailed instructions for users.
 
-You can use the `-v`/`--volume` or `--mount` flags to `docker run` to configure bind mounts of directories or files [@docker_use_2019], including options, as shown in the following examples.
+You can use the `-v`/`--volume` or preferably `--mount` flags to `docker run` to configure bind mounts of directories or files [@docker_use_2019], including options, as shown in the following examples.
 If the target path exists within the image, the bind mount will replace it for the started container.
 
 \footnotesize
 
 ```bash
 # mount directory
-docker run --volume /home/user/project:/project mycontainer
+docker run --mount type=bind,source=/home/user/project,target=/project mycontainer
```

Comment:
Any reason to not use envars here for $HOME?

Reply:
Done!